src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java

```diff
@@ -779,4 +779,25 @@
 
     }
 
+    @Override
+    public RepoGenericResponse<Elastigroup> importASG(ElastigroupImportASG elastigroupImportASG, String asgName, String dryRun, String region, String authToken, String account) {
```

Comment:
This will get changed if we remove ElastigroupImportASG . Please modify accordingly.